.travis.yml

```diff
@@ -14,7 +14,7 @@
 #
 language: objective-c
 sudo: false
-osx_image: xcode7.3
+osx_image: xcode8.2
```

Comment:
could we move this to another PR?

Reply:
This is to make travis work. Hao suggested me to update it to 8.3 actually.

Reply:
that's fine I am saying could we do this as a separate PR or did is it specifically tied to this pr?

Reply:
Hmm I tried 8.3 as well but travis keeps getting the error information of "/Users/travis/.travis/job_stages: line 166: shell_session_update: command not found".  Do you have any idea how to fix this?

Reply:
If we can find a way to make travis pass first, sure we can make a separate PR for this file change

Reply:
OK i've created https://github.com/google/EarlGrey/pull/676 to see if just that passes.